plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -85,11 +93,12 @@ public class FakerMetadata
     private final List<SchemaInfo> schemas = new ArrayList<>();
     private final double nullProbability;
     private final long defaultLimit;
+    private final FakerFunctionProvider functionsProvider;
 
     @GuardedBy("this")
     private final Map<SchemaTableName, TableInfo> tables = new HashMap<>();
-
-    private final FakerFunctionProvider functionsProvider;
+    @GuardedBy("this")
+    private final Map<SchemaTableName, ConnectorViewDefinition> views = new HashMap<>();
```

Comment:
Could you add `- [](sql-view-management)` to faker.md?

Reply:
https://github.com/trinodb/trino/pull/24581